src/main/java/seedu/address/logic/commands/NewtagCommand.java

```diff
@@ -26,6 +27,13 @@ public NewtagCommand(Tag tag) {
 
     @Override
     public CommandResult execute(Model model) {
-        return new CommandResult("Valid newtag Command Received: " + tag);
+        requireAllNonNull(model);
+        boolean success = model.addTag(tag);
```

Comment:
Should the boolean be named isSuccessful instead of success?

Reply:
Will change that!